cve_bin_tool/parsers/ruby.py

```diff
@@ -29,7 +73,9 @@ def run_checker(self, filename):
                 ):
                     product = line.strip().split()[0]
                     version = line.strip().split("(")[1][:-1]
-                    vendor = self.find_vendor(product, version)
-                    if vendor is not None:
-                        yield from vendor
+                    vendors = self.find_vendor(product, version)
+                    if vendors is not None:
+                        for v in vendors:
+                            self.generate_purl(product, version, v.product_info.vendor)
```

Comment:
Let's leave this out for now since we're not actually using the purl data for anything (yet)